.github/workflows/ci-dev-release.yml

```diff
@@ -49,6 +49,8 @@ jobs:
       - uses: actions/checkout@v4
         with:
           ref: ${{ inputs.ref }}
+      - name: Install CLI
+        run: SDKS='' make install
```

Comment:
Why is this necessary?  We already build the CLI for integration tests, and this won't include coverage data :thinking:

Reply:
We need the CLI to be able to run `get-job-matrix.py` and in this job we don't have the CLI yet.

Reply:
Ah I see, and this is literally only used for `get-job-matrix.py`, so code coverage isn't really a concern there.